app/src/main/java/com/bogdan/codeforceswatcher/features/actions/ActionsAdapter.kt

```diff
@@ -23,112 +18,29 @@ class ActionsAdapter(
     private val itemClickListener: (Int) -> Unit
 ) : RecyclerView.Adapter<ActionsAdapter.ViewHolder>() {
 
-    private var items: List<Action> = listOf()
+    private var items: List<ActionItem> = listOf()
 
     override fun getItemCount() = items.size
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val actionItem = R.layout.view_action_item
```

Comment:
`actionItemView` rather.

Reply:
And I meant to extract `LayoutInflater.from(context).inflate(actionItem, parent, false)` 😄